Nautilus/MonoBehaviours/EntitySpawner.cs

```diff
@@ -35,40 +35,18 @@ IEnumerator SpawnAsync()
             Destroy(gameObject);
             yield break;
         }
-        
-        if (!prefab.IsPrefab())
-        {
-            prefab.SetActive(false);
-        }
 
         GameObject obj = UWE.Utils.InstantiateDeactivated(prefab, spawnInfo.SpawnPosition, spawnInfo.Rotation, spawnInfo.ActualScale);
+        var lwe = obj.GetComponent<LargeWorldEntity>();
 
-        LargeWorldEntity lwe = obj.GetComponent<LargeWorldEntity>();
-
-        LargeWorldStreamer lws = LargeWorldStreamer.main;
-        yield return new WaitUntil(() => lws != null && lws.IsReady()); // first we make sure the world streamer is initialized
-
-        // non-global objects cannot be spawned in unloaded terrain so we need to wait
-        if (lwe is {cellLevel: not (LargeWorldEntity.CellLevel.Batch or LargeWorldEntity.CellLevel.Global)})
-        {
-            Int3 batch = lws.GetContainingBatch(spawnInfo.SpawnPosition);
-            yield return new WaitUntil(() => lws.IsBatchReadyToCompile(batch)); // then we wait until the terrain is fully loaded (must be checked on each frame for faster spawns)
-        }
-
-        LargeWorld lw = LargeWorld.main;
-
-        yield return new WaitUntil(() => lw != null && lw.streamer.globalRoot != null); // need to make sure global root is ready too for global spawns.
-            
-        lw.streamer.cellManager.RegisterEntity(obj);
+        if (lwe != null)
```

Comment:
Don't make having a `LargeWorldEntity` optional because the game has no idea how to save it and we cant assume that they're transient.

If an object is missing an LWE component, it's most likely that the user forgot to add it, so it's better to let them know about than to have an arbitrary fallback.